### ISSUE TITLE
"View all versions" toggle on search results page -- add labels

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#136]
From NYU A11y Review - 

**Toggle switch doesn't have a label.**

The toggle switch (checkbox) to turn on/off "All Versions" does not have a for/id association with the label/input.

Consider:
```
<div class="ui toggle checkbox">
   <input class="hidden" readonly="" tabindex="0" type="checkbox" value="" name="toggle" id="toggle">
   <label for="toggle">View all versions</label>
</div>
```

**Comment by lhenze** [comment]
Created PR: https://github.com/inveniosoftware/invenio-app-rdm/pull/1141
